playbooks/molecule/resources/xnat/verify.yml

```diff
@@ -16,3 +16,63 @@
           - response.status == 200
           - response.server == "nginx"
           - response.content is search('MIRSG XNAT')
+
+- name: Verify correct Docker version is installed
+  hosts: xnat_cserv
+  tasks:
+    - name: Get Docker server version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Server.Version}}'{% endraw %}"
+      changed_when: false
+      register: server_version
+
+    - name: Get Docker client version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Client.Version}}'{% endraw %}"
+      changed_when: false
+      register: client_version
+
+    - name: Check Docker server version
+      ansible.builtin.assert:
+        that:
+          - server_version.stdout is version("24.0.9")
+          - client_version.stdout is version("24.0.9")
```

Comment:
Hard coding this here as well as in the role feels dirty. Can these be made variables?

Reply:
there is an issue for this https://github.com/UCL-MIRSG/ansible-collection-infra/issues/68 but may as well do it in this pr?

Reply:
Oh, yeah (funny how I've completely forgotten about this stuff). Yeah, I reckon here is fine. Issue #68 may not fix both bits if addressed in the future.